arlo-client/src/components/Audit/Setup/useParticipantsApi.ts

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react'
+import { toast } from 'react-toastify'
+import { api } from '../../utilities'
+
+export type IJurisdictions = {
+  id: string
+  name: string
+}[]
+
+const useParticipantsApi = (electionId: string) => {
```

Comment:
Any reason this is named Participants rather than Jurisdictions? It was a bit confusing to me elsewhere